src/main/scala/org/holmesprocessing/totem/services/dnsmeta/dnsmeta.py

```diff
@@ -11,12 +11,18 @@
 import gatherdns
 from time import localtime, strftime
 
-# imports for services
-from holmeslibrary.services import ServiceConfig
+#imports for reading configuration file
+import json
+
+# reading configuration file
+def ServiceConfig(filename):
+    configPath = filename
+    # TODO : handle file not found exception
+    config = json.loads(open(configPath).read())
+    return config
 
-# Get service meta information and configuration
 Config = ServiceConfig("./service.conf")
-Config.dnsmeta.rdtypes = [item.strip() for item in Config.dnsmeta.rdtypes.split(',')]
+Config["dnsmeta"]["rdtypes"] = [item.strip() for item in Config["dnsmeta"]["rdtypes"].split(',')]
```

Comment:
since this is json you might be able to just have them input directly as a list and skip the list creation logic.